cpu/stm32_common/include/periph_cpu_common.h

```diff
@@ -43,10 +43,26 @@ extern "C" {
  * @brief   Available peripheral buses
  */
 enum {
-    APB1,
-    APB2
+    APB1,           /**< APB1 bus */
+    APB2            /**< APB2 bus */
 };
```

Comment:
actually no need to type this enum, as we are just interested in it's members, and typing the `bus` param as `uint8_t` gives us more control over the actual width of the enum value...